datemath/__init__.py

```diff
@@ -1,9 +1,18 @@
-from .helpers import parse, DateMathException
+from __future__ import annotations
 
-def dm(expr, **kwargs):
+from datetime import datetime
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from typing_extensions import Unpack
+    from arrow import Arrow
+
+from .helpers import ParseParams, parse as parse, DateMathException as DateMathException
+
+def dm(expr: str | int, **kwargs: Unpack[ParseParams]) -> Arrow:
```

Comment:
By using an unpacked `TypedDict` to annotate `kwargs`, you don't have to duplicate all your default keyword argument parameter's default values to have these functions fully typed.